src/strings.cpp

```diff
@@ -676,16 +679,17 @@ struct UnitsLong {
 
 /** Unit conversions for velocity. */
 static const Units _units_velocity[] = {
-	{ {   1,  0}, STR_UNITS_VELOCITY_IMPERIAL },
-	{ { 103,  6}, STR_UNITS_VELOCITY_METRIC   },
-	{ {1831, 12}, STR_UNITS_VELOCITY_SI       },
+	{ {    1,  0}, STR_UNITS_VELOCITY_IMPERIAL,     0 },
+	{ {  103,  6}, STR_UNITS_VELOCITY_METRIC,       0 },
+	{ { 1831, 12}, STR_UNITS_VELOCITY_SI,           0 },
+	{ {37888, 16}, STR_UNITS_VELOCITY_PLAYORIENTED, 1 },
```

Comment:
I know all the others are missing, but I think a comment explaining where these numbers came from would be a good idea

Reply:
Added